android/app/build.gradle

```diff
@@ -95,6 +95,8 @@ android {
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
+        testInstrumentationRunner "com.salesforce.samples.smartsyncexplorerreactnative.ReactNativeTestRunner"
```

Comment:
Why do we need a different test runner? Can't we use the standard one we use in all our other tests (see [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SmartStore/build.gradle#L39))?